website/docs/d/iam_policy_document.html.markdown

```diff
@@ -184,6 +186,10 @@ data "aws_iam_policy_document" "event_stream_bucket_role_assume_role_policy" {
       type        = "AWS"
       identifiers = ["${var.trusted_role_arn}"]
     }
+    principals {
```

Comment:
Nit: @andyjp94 do you mind adding a newline before this new `principals` block to keep the spacing between blocks consistent?

Reply:
sure I can do that

Reply:
I've made the changes now, let me know if that is what you expected

Reply:
looks great! thank you @andyjp94